### ISSUE TITLE
Able to set max simultaneous trade

### ISSUE THREAD

**Issue by HardenedSteel** [haveno-dex/haveno#1053]
A market maker should able set maximum simultaneous trade or auto-funding of trades. With this feature possible malicious arbitrator could rug less coins.

**Comment by woodser** [comment]
Offers are intentionally funded and verified so multiple offers cannot be created with the same funds.

**Comment by HardenedSteel** [comment]
I think you misunderstand me, This feature will limit amount of opened trades at same time. For an example if the user sets the limit 1 then if someone takes an offer from that user rest of the offers will be disabled.

As I stated above this will limit the trust to arbitrators.

**Comment by woodser** [comment]
The intention is that all offers on the books are fully funded, and won't disappear when some offers are taken.

**Comment by HardenedSteel** [comment]
this issue is for making it disappear in order to limit the trust to arbitrators.

**Comment by woodser** [comment]
It would enable a fractional reserve, where people can post offers to sell xmr they don't have.

**Comment by HardenedSteel** [comment]
that's the case with #1055 this issue is different. The limit is about how many trades are opened, not how much XMR is traded.

**Comment by woodser** [comment]
Ah I see, so a user could post 20 offers, but only be able to have 5 trades active at a time for example.

**Comment by HardenedSteel** [comment]
yes correct, the idea is minimizing the risk by not putting the all funds in once

**Comment by boldsuck** [comment]
You can enable or disable individual or all offers in 'MY OPEN OFFERS'.

**Comment by monerobull** [comment]
I like this idea.
Example: 
I make 20 offers, all funded. I have max-open-trades set to 5. 
In retos case there are currently 2 arbs, if one goes rogue, they on average get 2.5 offers assigned to them instead of 10. Sure, they could quickly close out their 2.5 offers and start new ones, but now they only get an average 1.5 assigned to them and 0.75 on the next round. Eventually, all 5 max-open-trades would be assigned to the non-rogue arbitrator and the rogue one couldn't get any new offers from me. Once there are more arbitrators, this effect becomes even greater.
